modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -0,0 +1,158 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+:_content-type: PROCEDURE
+[id="ossm-integrating-with-user-workload-monitoring_{context}"]
+= Integrating with user-workload monitoring
+
+By default, {SMProductName} (OSSM) installs the Service Mesh control plane (SMCP) with a dedicated instance of Prometheus for collecting metrics from a mesh. However, production systems need more advanced monitoring systems, like {product-title} monitoring for user-defined projects.
+
+The following steps show how to integrate Service Mesh with user-workload monitoring.
+
+.Prerequisites
+
+* User-workload monitoring is enabled.
+* {SMProductName} Operator 2.4 is installed.
+
+.Procedure
+
+. Configure the SMCP for external Prometheus:
++
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  addons:
+    prometheus:
+      enabled: false # <1>
+    grafana:
+      enabled: false # <2>
+    kiali:
+      enabled: false
+----
+<1> Disable the default Prometheus instance provided by OSSM.
+<2> Disable Grafana. It is not supported with an external Prometheus instance.
+
+. Apply a custom network policy to allow ingress traffic from the `user-workload monitoring` namespace:
```

Comment:
Is `user-workload monitoring` the actual name of the namespace, or should this be "`user-workload' monitoring namespace"? There's no literal reference to the namespace in the YAML, so it's hard to tell, but IIRC I haven't ever seen a namespace with a space in the name, it's usually dashes I think

Reply:
Looking at https://docs.openshift.com/container-platform/4.12/monitoring/enabling-monitoring-for-user-defined-projects.html I now wonder if it's `openshift-user-workload-monitoring` so I will chechk.

Reply:
The monitoring namespace is selected via a label selector (`network.openshift.io/policy-group: monitoring`), not via a direct reference, so it may be more accurate to just call it the "monitoring namespace". Currently, I believe at least the namespaces `openshift-monitoring` and `openshift-user-workload-monitoring` have this label. I _want_ to say the only namespace that _needs_ to access the workload namespace (`bookinfo` in this case) would be `openshift-user-workload-monitoring`, in which case it would be accurate to just say `openshift-user-workload-monitoring`, but I'm not 100% sure on that.

Either way, I think just referring to it as the "monitoring namespace" here would be better.
```suggestion
. Apply a custom network policy to allow ingress traffic from the monitoring namespace:
```